common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -246,17 +246,26 @@ class @Problem
 
   # Scroll to problem metadata and next focus is problem input
   scroll_to_problem_meta: =>
-    @questionTitle = @$(".problem-header")
-    if @questionTitle.length > 0
+    questionTitle = @$(".problem-header")
```

Comment:
Note to reviewers: there is no reason questionTitle can't just be a local variable.